src/test/groovy/ee/tuleva/onboarding/account/AccountStatementControllerSpec.groovy

```diff
@@ -28,8 +32,56 @@ class AccountStatementControllerSpec extends BaseControllerSpec {
         1 * accountStatementService.getAccountStatement(_ as Person) >> fundBalances
         1 * fundTransferStatisticsService.saveFundValueStatistics(fundBalances, statisticsIdentifier)
         expect:
-            mockMvc.perform(get("/v1/pension-account-statement").header("x-statistics-identifier", statisticsIdentifier))
-                .andExpect(status().isOk())
+        mockMvc.perform(get("/v1/pension-account-statement").header("x-statistics-identifier", statisticsIdentifier))
+            .andExpect(status().isOk())
+    }
+
+    def "/pension-account-statement endpoint accepts language header and responds with appropriate fund.name"() {
+        given:
+        List<FundBalance> fundBalances = AccountStatementFixture.sampleConvertedFundBalanceWithActiveTulevaFund
+
+        UUID statisticsIdentifier = UUID.randomUUID()
+        1 * accountStatementService.getAccountStatement(_ as Person) >> fundBalances
+
+        expect:
+        mockMvc.perform(get("/v1/pension-account-statement")
+            .header("x-statistics-identifier", statisticsIdentifier)
+            .header("Accept-Language", language)
+        )
+            .andDo(MockMvcResultHandlers.print())
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+            .andExpect(jsonPath('$[0].fund.name', is(translation)))
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+        where:
+        language | translation
+        "et"     | "Tuleva maailma aktsiate pensionifond"
+        "en"     | "Tuleva world stock pensionfund"
+
+    }
+
+    def "/pension-account-statement endpoint falls back to Estonian if no Language-Accept given"() {
+        given:
+        List<FundBalance> fundBalances = AccountStatementFixture.sampleConvertedFundBalanceWithActiveTulevaFund
+
+        UUID statisticsIdentifier = UUID.randomUUID()
+        1 * accountStatementService.getAccountStatement(_ as Person) >> fundBalances
+
+        expect:
+        mockMvc.perform(get("/v1/pension-account-statement")
+            .header("x-statistics-identifier", statisticsIdentifier)
+        )
+            .andDo(MockMvcResultHandlers.print())
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+            .andExpect(jsonPath('$[0].fund.name', is(translation)))
+            .andExpect(jsonPath('$', hasSize(fundBalances.size())))
+        where:
+        language | translation
+        "et"     | "Tuleva maailma aktsiate pensionifond"
+
```

Comment:
Duplicate test. You could add a line to the previous test like:
` null     | "Tuleva maailma aktsiate pensionifond"`
that would test the same scenario